src/modules/common-elements/switch-labels-group.styles.less

```diff
@@ -49,6 +49,12 @@
     }
   }
 
+  // TODO: Circle back and remove this when SIDE NAV is removed
+  // Temporary quick fix for sizing issues between 1281 - 1420
+  @media (min-width: 1281px) and (max-width: 1420px) {
```

Comment:
make a breakpoint for this, call it temp or something so we know to fix it later, remove TODO